src/components/VerificationRequest/VerificationRequestList.tsx

```diff
@@ -1,54 +1,160 @@
-import React from "react";
-import BaseList from "../List/BaseList";
-import { Col, Row } from "antd";
+import React, { useState, useEffect, useCallback } from "react";
 import { useTranslation } from "next-i18next";
 import { useAtom } from "jotai";
 import { currentNameSpace } from "../../atoms/namespace";
 import verificationRequestApi from "../../api/verificationRequestApi";
-import VerificationRequestCard from "./VerificationRequestCard";
 import AletheiaButton from "../Button";
+import { Grid } from "@mui/material";
+import {
+    GridActionsCellItem,
+    GridColDef,
+    GridRowParams,
+} from "@mui/x-data-grid";
+import colors from "../../styles/colors";
+import PaginatedDataGrid from "../PaginetedDataGrid";
 
 const VerificationRequestList = () => {
     const { t } = useTranslation();
     const [nameSpace] = useAtom(currentNameSpace);
+    const [verificationRequests, setVerificationRequests] = useState([]);
+
+    const truncateWithEllipsis = (value, maxLength) => {
+        if (!value) return "";
+        const truncatedValue = value.substring(0, maxLength);
+        return value.length > maxLength
+            ? `${truncatedValue}...`
+            : truncatedValue;
+    };
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await verificationRequestApi.get({});
```

Comment:
chore: create an issue to add verification requests's query pagination

Reply:
#1355